includes/payments/functions.php

```diff
@@ -851,54 +852,46 @@ function give_get_total_sales() {
  *
  * @since  1.0
  *
+ * @param bool $recalculate Recalculate earnings forcefully.
+ *
  * @return float $total Total earnings.
  */
-function give_get_total_earnings() {
-
-	$total = get_option( 'give_earnings_total', false );
+function give_get_total_earnings( $recalculate = false ) {
```

Comment:
@DevinWalker I think it will be good if we provide a param to recalculate total forcefully